testing/integration/upgrade_test.go

```diff
@@ -48,6 +52,12 @@ agent.upgrade.watcher:
   crash_check.interval: 15s
 `
 
+// notable versions used in tests
+var version_8_2_0 = version.NewParsedSemVer(8, 2, 0, "", "")
+var version_8_6_0 = version.NewParsedSemVer(8, 6, 0, "", "")
+var version_8_7_0 = version.NewParsedSemVer(8, 7, 0, "", "")
+var version_8_9_0_SNAPSHOT = version.NewParsedSemVer(8, 9, 0, "SNAPSHOT", "")
```

Comment:
It would be really nice IMO to comment for each of these versions why they are special.  For example, `version_8_2_0` is special because versions starting from that one support the `--non-interactive` install flag, etc.